### ISSUE TITLE
Problem with .NET Core SDK v3.1

### ISSUE THREAD

**Issue by wallymathieu** [AArnott/CodeGeneration.Roslyn#180]
I've tried out the tool on a non windows system, however, the errors reported are from the msbuild target:
```
/home/runner/.nuget/packages/codegeneration.roslyn.buildtime/0.6.1/build/CodeGeneration.Roslyn.BuildTime.targets(73,5): error CGR1001: CodeGeneration.Roslyn.Tool (dotnet-codegen) is not available, code generation won't run. Please check https://github.com/AArnott/CodeGeneration.Roslyn for usage instructions. [/home/runner/work/with/with/src/Timing/Timing.csproj]
 ```

see for instance build logs from previous builds:

https://github.com/wallymathieu/with/runs/337456154
https://travis-ci.org/wallymathieu/with/builds/621803654?utm_source=github_status&utm_medium=notification

**Comment by amis92** [comment]
Hi. It's not a non-Windows problem. It's a problem of SDK v3.1 not containing v2.1 runtime, and on the other hand, dotnet-codegen not having proper RollForward declaration. This is going to be fixed in 0.7 via #178.

For now, you can either (on the cloud build):
* manually install runtime v2.1 (which you probably already have on your desktop, but is not available when you do `setup-dotnet` action with 3.1.100 sdk;
* declare environment variable: `DOTNET_ROLL_FORWARD=Major`
* wait for 0.7 which will Just Work™

**Comment by wallymathieu** [comment]
Thanks! That clarifies things!
